bin/download

```diff
@@ -34,13 +36,18 @@ download_golang () {
 
     platform=$(get_platform)
     arch=$(get_arch)
-    download_url="https://dl.google.com/go/go${version}.${platform}-${arch}.tar.gz"
+    if [ -n $RTX_GOLANG_DOWNLOAD_MIRROR ]; then
```

Comment:
this and RTX_TRACE will fail if they are not set because `set -u` is set on this file, you need to do something like this to default the values:

```suggestion
    if [ -z "${RTX_GOLANG_DOWNLOAD_MIRROR:-}" ]; then
```